### DISCUSSION TITLE
Hass Notification just like in Frigate Events

### DISCUSSION THREAD

**Discussion by bortek** [blakeblackshear/frigate#8070] [General]
Hi

I have been struggling to get notifications from Hass Frigate integration. In Frigate I have everything setup as I wanted and using required zones. The events generated there are (almost) perfect without false positives. 

What I simply want now is that my Frigate Integration in Home Assistant to send me notifications of completed (not in progress) events and that should reflect events generated in the Frigate GUI->Events. 

How do I create an automation that will do it?

I tried to use some examples from https://github.com/blakeblackshear/frigate/discussions/2898 to no avail. I get instead lots of events and thus many notifications. Isn't there a simple way to achieve it?

**Comment by NickM-27** [comment]
you would use #2898 and just alert when `type == "end"`. If you are using zones then you'll need to filter for that as well
